packages/ui-extensions/docs/surfaces/admin/staticPages/admin-extensions.doc.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable no-template-curly-in-string */
+import {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const data: LandingTemplateSchema = {
+  title: 'Admin App Extensions',
+  description:
+    'Admin App Extensions are a way to extend the Shopify Admin experience. The extensions can be used to add new functionality to the Shopify Admin. It allows you to integrate your app into the Shopify Admin in a more meaningful way.',
+  image: '/assets/landing-pages/templated-apis/hero.png',
+  darkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  tabletImage: '/assets/landing-pages/templated-apis/hero.png',
+  tabletDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  mobileImage: '/assets/landing-pages/templated-apis/hero.png',
+  mobileDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  id: 'admin-extensions',
+  sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'actionExtensions',
+      title: 'Overview',
+      image: 'action-extension-example.gif',
+      sectionContent: 'Extend the Shopify Admin with App Extensions.',
+      sectionNotice: [
+        {
+          title: 'Developer Preview',
+          sectionContent:
+            'Please not that Admin Action and Block Extensions are currently in a Developer Preview. You may on see them while developing locally on a development store.',
+          type: 'warning',
+        },
+      ],
+      sectionCard: [
+        {
+          name: 'Getting Started',
+          subtitle: 'Get started building your first Admin Extension.',
```

Comment:
The subtitle copy is meant to be more of a description of the type of content you're linking to 

<img width="563" alt="Screenshot 2023-07-13 at 12 14 25 PM" src="https://github.com/Shopify/ui-extensions/assets/1638215/87c23e7e-9c44-4b0a-818a-8df78c6c078f">

-----------------------

**See Hydrogen for a great example of how to structure these;** 

https://shopify.dev/docs/api/hydrogen-react


<img width="540" alt="Screenshot 2023-07-13 at 12 17 44 PM" src="https://github.com/Shopify/ui-extensions/assets/1638215/87b830f8-6204-4bac-a9b2-42feb55a55ce">
<img width="532" alt="Screenshot 2023-07-13 at 12 17 40 PM" src="https://github.com/Shopify/ui-extensions/assets/1638215/50cb2739-213a-43de-b302-cd8fbfc1dfb9">

Reply:
Still might be a bit wordy. Any thoughts?
<img width="621" alt="Screenshot 2023-07-13 at 6 44 09 PM" src="https://github.com/Shopify/ui-extensions/assets/390744/287bef97-ee75-413a-8000-a69cfb111afe">

Reply:
Updated